lib/checks/aria/required-parent.js

```diff
@@ -62,26 +79,26 @@ function getAriaOwners(element) {
 	return owners.length ? owners : null;
 }
 
-var missingParents = getMissingContext(virtualNode);
-
-if (!missingParents) {
-	return true;
-}
+/**
+ * Recursively attempt to find parent with a desired role
+ * @param {HTMLElement} elm element
+ * @param {String} role aria role
+ * @returns {Boolean}
+ */
+function hasParentWithRole(elm, role) {
+	const parent = dom.getComposedParent(elm);
```

Comment:
Wouldn't it be easier to pass `virtualNode` into the function? Then you wouldn't have to call `getComposedParent` and could instead use `virtalNode.parent` which is already the composed parent.